tests/functional/arxiv/fixtures/http_server/records/arxiv-physics-hep-ex.xml

```diff
@@ -10,18 +10,19 @@
  <setSpec>physics:hep-ex</setSpec>
 </header>
 <metadata>
- <arXiv xmlns="http://arxiv.org/OAI/arXiv/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://arxiv.org/OAI/arXiv/ http://arxiv.org/OAI/arXiv.xsd">
- <id>1706.05371</id><created>2017-06-16</created><updated>2017-12-12</updated><authors><author><keyname>Anderson</keyname><forenames>Tyler</forenames></author><author><keyname>Cutter</keyname><forenames>Jacob</forenames></author><author><keyname>Dhaliwal</keyname><forenames>Navneet</forenames></author><author><keyname>Dalager</keyname><forenames>Olivia</forenames></author><author><keyname>Godfrey</keyname><forenames>Benjamin</forenames></author><author><keyname>Hillbrand</keyname><forenames>Seth</forenames></author><author><keyname>Irving</keyname><forenames>Michael</forenames></author><author><keyname>Manalaysay</keyname><forenames>Aaron</forenames></author><author><keyname>Montoya</keyname><forenames>Juan</forenames></author><author><keyname>Morad</keyname><forenames>James</forenames></author><author><keyname>Neher</keyname><forenames>Christian</forenames></author><author><keyname>Stolp</keyname><forenames>Dustin</forenames></author><author><keyname>Tripathi</keyname><forenames>Mani</forenames></author></authors><title>Evaluation of Silicon Photomultipliers for use as Photosensors in Liquid
-  Xenon Detectors</title><categories>physics.ins-det hep-ex</categories><comments>10 pages, 9 figures</comments><license>http://arxiv.org/licenses/nonexclusive-distrib/1.0/</license><abstract>  Silicon photomultipliers (SiPMs) are potential solid-state alternatives to
-traditional photomultiplier tubes (PMTs) for single-photon detection. In this
-paper, we report on evaluating SensL MicroFC-10035-SMT SiPMs for their
-suitability as PMT replacements. The devices were successfully operated in a
-liquid-xenon detector, which demonstrates that SiPMs can be used in noble
-element time projection chambers as photosensors. The devices were also cooled
-down to 170 K to observe dark count dependence on temperature. No dependencies
-on the direction of an applied 3.2 kV/cm electric field were observed with
-respect to dark-count rate, gain, or photon detection efficiency.
-</abstract></arXiv>
```

Comment:
what is the response from arXiv? If there are new lines there, there should be newlines here, if not there shouldn't. You should not mess with the response in any case.